src/data/events.js

```diff
@@ -39,3 +39,97 @@ export const pastEvents = [
     image: '/assets/event-image.png',
   },
 ];
+
+export const values = [
+  {
+      icon:"/icon6.jpeg",
```

Comment:
to avoid any future issues, create a members.js file that renders data for the About page instead of using the same file for Events data

Reply:
Working on it